pubspec.yaml

```diff
@@ -9,6 +9,12 @@ dependencies:
   http: '>=0.12.0 <1.0.0'
   flutter:
     sdk: flutter
+
+  platform: ^2.0.0
```

Comment:
Could you guide me on what this is for? I've no idea about this. I hope you can explain this to me..

Reply:
Ok,
This plugin is https://pub.dartlang.org/packages/platform. 
It is written in this README, but if you use static properties, when you build on PC(not android or iOS), you can not mock them, so it becomes state that platform not supported.
You can use this plugin to mock it and make it testable.